docs/user-guide/nodes.rst

```diff
@@ -113,44 +113,29 @@ To install *Nodes* child support, follow the instructions below.
 
  $ sudo yum install pulp-nodes-child pulp-nodes-consumer-extensions pulp-agent
 
-2. The communication between the child and parent nodes is secured using OAuth. The child node
-   must have OAuth enabled and configured. Please see :ref:`OAuth <oauth-config>` for instructions
-   on enabling and configuring OAuth.
+2. Configure ``/etc/pulp/server.conf`` on the child node to connect to the parent node via OAuth.
+   The communication between the child and parent nodes is secured using OAuth. Add the following to
+   the ``/etc/pulp/server.conf`` file on the child node.
 
 ::
 
  [oauth]
- enabled: true
- oauth_key: Xohkaethaama5eki
- oauth_secret: eePa7Bi3gohdir1pai2icohvaidai0io
-
-.. warning::
-
-   Do not use the key or secret given in the above example. It is important that you use unique and
-   secret values for these configuration items.
+ user_id:  <EDIT: admin user on parent node>
 
-3. Edit ``/etc/pulp/nodes.conf`` on the child node and set the parent OAuth *key* and *secret* to
-   match values found in ``/etc/pulp/server.conf`` on the parent node. The *user_id* must be updated
-   as needed to match a user with administration privileges on the parent node.
```

Comment:
The example here is good but by removing the description which includes editing `/etc/pulp/nodes.conf' results in incorrectly instructing the user to edit `/etc/pulp/server.conf` for the `[parent_node]` settings.